src/debugger/main.cpp

```diff
@@ -0,0 +1,21 @@
+#include <iostream>
+
```

Comment:
`#include <string>`. I'm assuming it's not broken since it's included in one of the other headers you use but best practice is include what you use?

Reply:
Good catch but this is just a skeleton to remind me to fix this